pkg/controller/postgres/postgres_controller.go

```diff
@@ -194,6 +203,11 @@ func (r *ReconcilePostgres) Reconcile(request reconcile.Request) (reconcile.Resu
 			return reconcile.Result{Requeue: true, RequeueAfter: p.GetReconcileTime(instance)}, nil
 		}
 
+		annotations.Add(instance, annotations.Created, annotations.Created)
+		if err := r.client.Update(ctx, instance); err != nil {
+			return reconcile.Result{}, err
+		}
+
 		// return the connection secret
 		if err := r.resourceProvider.ReconcileResultSecret(ctx, instance, ps.DeploymentDetails.Data()); err != nil {
```

Comment:
This "ReconcileResultSecret" will now not happen if the "created" annotation exists (or the pre-existing case where `skipCreate` is specified on the spec. I imagine this is fine (`ps` won't exist here unless we do a create (unless there's another function that would "get" than existing one?), but I thought I'd highlight it anyway.

Reply:
i would imagine this is okay, our alerts will highlight if anything is wrong. And for the most part after a restore the secret should get updated.